cpp/include/cudf/predicates.hpp

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
I agree that this could probably move to `sort.hpp` and it would still make sense.

Reply:
@harrism Any objections to moving `cudf::is_sorted` from `predicates.hpp` to `sorting.hpp` (specifically: as part of this PR)?

Reply:
Nope. I assume there are no other predicates?

Reply:
Correct.